src/pykeen/typing.py

```diff
@@ -104,6 +109,12 @@ class ScorePack(NamedTuple):
 
 Sign = Literal[-1, 1]
 
+#: the prediction and training mode
+Mode = Literal["training", "validation", "testing"]
+TRAINING: Mode = "training"
+VALIDATION: Mode = "validation"
```

Comment:
Can we make it clearer that this is the *inductive*  validation mode? For transductive validation, we would still need to pass `None`.

Reply:
huh, that's actually a good idea

Reply:
I have my moments ;)